log/log.go

```diff
@@ -263,8 +273,8 @@ func (w *stdoutWriter) logAtLevel(level syslog.Priority, msg string) {
 	}
 }
 
-func (log *impl) auditAtLevel(level syslog.Priority, msg string) {
-	text := fmt.Sprintf("%s %s", auditTag, msg)
+func (log *impl) auditAtLevel(level syslog.Priority, msg string, a ...interface{}) {
+	text := fmt.Sprintf("%s %s", auditTag, fmt.Sprintf(msg, a...))
```

Comment:
What about
```suggestion
	msg := fmt.Sprintf("%s %s", auditTag, msg)
```
and then calling `log.w.logAtLevel(level, msg, a...)`?